sources/packages/backend/apps/db-migrations/src/sql/Announcements/Create-announcements-table.sql

```diff
@@ -0,0 +1,34 @@
+CREATE TABLE IF NOT EXISTS sims.announcements (
+    id SERIAL PRIMARY KEY,
+    message_title VARCHAR(100) NOT NULL,
+    message VARCHAR(200) NOT NULL,
+    start_date TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW (),
+    end_date TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW (),
+    target VARCHAR(200) NOT NULL,
```

Comment:
Isn't this supposed to be an array type?

Reply:
Yes that was mentioned in technical. 
![image](https://github.com/user-attachments/assets/de050537-3407-45c3-b9f1-909e9417465c)

Reply:
updated